src/compas_view2/__init__.py

```diff
@@ -37,3 +37,12 @@ def new_util_find_library(name):
 
 
 __all__ = ["HOME", "DATA", "DOCS", "TEMP", "DATA_OBJECT", "register"]
+
+
+def _find_resource(filename):
+    filename = filename.strip("/")
+    return os.path.abspath(os.path.join(DATA, filename))
+
+
+def get(filename):
+    return _find_resource(filename)
```

Comment:
Can be combined here. And let's keep __all__ at bottom

Reply:
Done! Just out of curiosity: why the `all` is always at the bottom? is it conventional or functional?

Reply:
just a convention